mne/preprocessing/tests/test_ecg.py

```diff
@@ -16,7 +16,8 @@
 
 def test_find_ecg():
     """Test find ECG peaks."""
-    raw = read_raw_fif(raw_fname)
+    # Test if ECG analysis will work on data that is not preloaded
+    raw = read_raw_fif(raw_fname, preload=False)
```

Comment:
True, but my worry is that if the preload default is changed one day, then the test will not pick up if `create_ecg_epochs` fails to load the data.

Do you still want me to make the change?